server/sockets.js

```diff
@@ -41,26 +41,28 @@ if (cluster.isMaster) {
 			// console.log('master received: ' + data);
 			switch (data.charAt(0)) {
 			case '*': {
-				// *socketid, ip, protocol
+				// *socketid, ip, protocol, connectedAt
```

Comment:
Is this really necessary? You can just use `Date.now` instead of sending `connectedAt` over IPC.

Reply:
The time would be less accurate if I did that because of the overhead of IPC and the work done in `sockets.js` before sending information about sockets to the main process.

Reply:
Your threshold is seven days. A few milliseconds isn't going to affect it at all. I don't see that as worth complicating the IPC protocol for.

Reply:
There are no results for bots when I search for `lastConnected` and `play.pokemonshowdown.com`, so I think it should be OK to change the property name without needing to do anything hacky to keep `lastDisconnected` named `lastConnected` in `|userdetails|`.

Reply:
You marked this as resolved but you've neither changed this nor responded...

Reply:
It is changed, I just forgot to update the comment.